client/stylesheets/_toCat.scss

```diff
@@ -3705,6 +3769,16 @@ On click the images will be displayed at normal size to complete the effect
     text-align: center !important;
 }
 
+.profileUserMore li a i {
+	font-size: 4rem;
+	color: #00bfff;
```

Comment:
Color literals like `#00bfff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs